e2e/tests/events.bats

```diff
@@ -90,7 +87,7 @@ load variables
 
     run_with_retry "dispatch get function ${func_name} -o json | jq -r .status" "READY" 8 5
 
-    run dispatch create eventdrivertype ticker kars7e/timer:latest
+    run dispatch create eventdrivertype ticker zimengyvmware/ticker:latest
```

Comment:
Isn't there a ticker available at dispatchframework now?

Reply:
yeah there is, use `dispatchframework/ticker:0.0.1`